htdocs/expedition/index.php

```diff
@@ -256,6 +280,8 @@
 		}
 
 		print "</table></div><br>";
```

Comment:
Closing the table is inside a if bu opening was outside. This is surely a bug.

Reply:
I haven't check the complete cope, only my changes, but you are right, i will fix it